index.js

```diff
@@ -164,6 +164,15 @@ exports.kinesisHandler = (records, opts = {}, context, callback) => {
       }
 
       if (error.name === 'HoldRequestConsumerError') {
+        if (error.errorType === 'empty-filtered-records') {
+          logger.warning(
```

Comment:
Just logging semantics here, not would `notice` be more appropriate? FWIW, here's an interpretation of the levels:

`warning`: May indicate that an error will occur if action is not taken
`notice`: Events that are unusual, but not error conditions.